tests/topotests/bgp_ipv4_over_ipv6/rfc5549_ebgp_ecmp.json

```diff
@@ -0,0 +1,205 @@
+{
+    "ipv4base": "10.0.0.0",
+    "ipv4mask": 24,
+    "ipv6base": "fd00::",
+    "ipv6mask": 64,
+    "ipv6base-link-local": "fe80::",
+    "link_ip_start": {"ipv4": "10.0.0.0", "v4mask": 24, "ipv6": "fd00::", "v6mask": 64},
+    "lo_prefix": {"ipv4": "1.0.", "v4mask": 32, "ipv6": "2001:DB8:F::", "v6mask": 128},
+    "routers": {
+        "r0": {
+            "links": {
+                "lo": {"ipv4": "auto", "type": "loopback"},
+                "r1-link1": {"ipv4": "auto"},
+                "r1-link2": {"ipv4": "auto"},
+                "r1-link3": {"ipv4": "auto"},
+                "r1-link4": {"ipv4": "auto"},
+                "r1-link5": {"ipv4": "auto"},
+            }
+        },
+        "r1": {
+            "links": {
+                "lo": {"ipv4": "auto", "type": "loopback"},
+                "r0-link1": {"ipv4": "auto"},
+                "r0-link2": {"ipv4": "auto"},
+                "r0-link3": {"ipv4": "auto"},
+                "r0-link4": {"ipv4": "auto"},
+                "r0-link5": {"ipv4": "auto"},
+                "r2-link0": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link1": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link2": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link3": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link4": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link5": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link6": {"ipv4": "auto", "ipv6": "auto"},
+                "r2-link7": {"ipv4": "auto", "ipv6": "auto"},
+            },
+            "bgp": {
+                "local_as": "100",
+                "default_ipv4_unicast": "False",
+                "address_family": {
+                    "ipv6": {
+                        "unicast": {
+                            "neighbor": {
+                                "r2": {
+                                    "dest_link": {
+                                        "r1-link0": {
+                                            "capability": "extended-nexthop",
```

Comment:
am I reading this right and you are explicitly calling out the extended-nexthop capability to the cli?  Why is this necessary?  Can you point me at any tests you have for something like `neighbor <interface> remote-as external`?

Reply:
Answers to the query : 
1. Yes, we are explicitly calling out the extended-nexthop capability to the cli.
This translates to command "neighbor X::X capability extended-nexthop" inside bgp.

2. This is done for bgp to carry ipv4 routes over ipv6 bgp session.

3. We don't have any test with neighbour configure with interface, all the tests are with neighbour X::X (ipv6 address).